recipe/meta.yaml

```diff
@@ -31,11 +28,92 @@ build:
+    - find source -type f -exec grep -Il '#!/usr/bin/perl -w' {} + | xargs sed -i 's|#!/usr/bin/perl -w|#!/usr/bin/env perl\nuse warnings;|g'
 
     # Need to use -a to preserve symlinks
     - cp -a source $PREFIX/MG5_aMC
 
+    # Use packaged versions of vendored code
+    ## TODO: CutTools
+    # For now use the vendored version
+    # c.f. https://github.com/conda-forge/mg5amcnlo-feedstock/issues/3#issuecomment-2660481386
+
+    # Compile the vendored CutTools
+    # c.f. https://github.com/conda-forge/mg5amcnlo-feedstock/issues/3#issuecomment-2660481386
+    # Provides $PREFIX/MG5_aMC/vendor/CutTools/includects/libcts.a
+    # c.f. https://github.com/mg5amcnlo/mg5amcnlo/blob/3d6799de1bcabee5173e8a3711f657a5cd5940cf/Template/NLO/Source/makefile#L73
+    - cd $PREFIX/MG5_aMC/vendor/CutTools
+    - echo -e "\n# Compiling CutTools"
+    - make FC=$FC FFLAGS="-fPIC -fno-automatic -O2 -funroll-all-loops $FFLAGS"
+    # Running 'make clean' deletes the files we want, so manually clean up
+    - find . -name "*.o" -exec rm {} +
+    - cd -
+
+    ## IREGI
+    # TODO: Compare https://github.com/mg5amcnlo/mg5amcnlo/blob/3.x/vendor/IREGI/src/oneloop/avh_olo_foriregi.f90 against avh_olo.f90 in oneloop
+    - rm -rf $PREFIX/MG5_aMC/vendor/IREGI
+    - mkdir -p $PREFIX/MG5_aMC/vendor/IREGI/src
+    - ln -s $PREFIX/lib/libiregi.a $PREFIX/MG5_aMC/vendor/IREGI/src/
+    - for file in $PREFIX/include/iregi/*; do ln -s "$file" $PREFIX/MG5_aMC/vendor/IREGI/src/; done
+    - mkdir -p $PREFIX/MG5_aMC/vendor/IREGI/src/qcdloop/ff
+    - ln -s $PREFIX/lib/libff.a $PREFIX/MG5_aMC/vendor/IREGI/src/qcdloop/ff/
+    - mkdir -p $PREFIX/MG5_aMC/vendor/IREGI/src/qcdloop/ql
+    - ln -s $PREFIX/lib/libqcdloop.a $PREFIX/MG5_aMC/vendor/IREGI/src/qcdloop/ql/
+
+    # Compile the vendored StdHEP
+    # TODO: Package this
+    # c.f. https://github.com/mg5amcnlo/mg5amcnlo/blob/3d6799de1bcabee5173e8a3711f657a5cd5940cf/Template/NLO/Source/makefile#L99
+    - cd $PREFIX/MG5_aMC/vendor/StdHEP
+    # Patch makefile to ensure conda-forge FFLAGS and CFLAGS are used
+    - sed -i "s|FFLAGS += -std=legacy|FFLAGS += -std=legacy -ffixed-line-length-none $FFLAGS|g" makefile
+    - sed -i "s|CFLAGS += -Wno-implicit-function-declaration|CFLAGS += -Wno-implicit-function-declaration $CFLAGS|g" makefile
+    - echo -e "\n# Compiling StdHEP"
+    - make all FC=$FC F77=$FC CC=$CC --jobs=$CPU_COUNT
+    - make clean
+    - cd -
+
+    # Symlink libraries to HEPTools/ directory tree as mg5amcnlo looks there
+    - mkdir -p $PREFIX/MG5_aMC/HEPTools/lib
+    - mkdir -p $PREFIX/MG5_aMC/HEPTools/include
+    ## OneLOop
+    - mkdir -p $PREFIX/MG5_aMC/HEPTools/oneloop
+    - ln -s $PREFIX/lib/libavh_olo.a $PREFIX/MG5_aMC/HEPTools/oneloop/
+    - ln -s $PREFIX/lib/libavh_olo.a $PREFIX/MG5_aMC/HEPTools/lib/
+    - for file in $PREFIX/include/oneloop/*; do ln -s "$file" $PREFIX/MG5_aMC/HEPTools/oneloop/; ln -s "$file" $PREFIX/MG5_aMC/HEPTools/include/; done
+  {% if (linux and (x86_64 or ppc64le)) %}
```

Comment:
Need to update https://github.com/conda-forge/ninja-hep-ph-feedstock/ to still build macOS and `linux-aarch64` outputs of `ningja-hep-ph` given that trying to do a build in this CI fails for macOS and upstream `mg5amcnlo` just disables `libquadmath` for those builds: https://github.com/mg5amcnlo/HEPToolsInstallers/blob/d56fe6c04242360076be533a653c187493ef1187/installNinja.sh#L21

c.f. https://github.com/conda-forge/ninja-hep-ph-feedstock/issues/6